docs/build-repro.md

```diff
@@ -127,3 +127,38 @@ jq '.' metadata.json
   "containerimage.digest": "sha256:..."
 }
 ```
+
+### Reproducing the pinned dependencies
+
+<!-- TODO: s/master/v0.11/ after the release -->
+Reproducing the pinned dependencies is supported in the master branch of BuildKit.
+
+e.g.,
+```bash
+buildctl build --frontend dockerfile.v0 --local dockerfile=. --local context=. --source-policy-file Dockerfile.pol
+```
+
+An example `Dockerfile.pol`:
```

Comment:
Is this old, as this seems to use the old format while I think it seems to be directly parsed into the new proto definition?

The old buildinfo format is deprecated and replaced by the SLSA provenance.

Reply:
Yep, fixed.